workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -68,6 +70,28 @@ private IInjectable Inject(MonoBehaviour behaviour, InjectableId injectableId, E
             return injectable;
         }
 
+        public void DisposeAllRequiredFields(MonoBehaviour behaviour)
+        {
+            var behaviourType = behaviour.GetType();
+            EnsureLoaded(behaviourType);
+            foreach (var fieldsToComponents in fieldInfoCache[behaviourType])
+            {
+                var fields = fieldsToComponents.Value;
+                foreach (var field in fields)
+                {
+                    var requirableToBeDisposed = field.GetValue(behaviour) as RequirableBase;
+                    if (requirableToBeDisposed == null)
+                    {
+                        logger.HandleLog(LogType.Error, new LogEvent(RequirableFieldDoesNotInheritRequirableBase)
+                            .WithField("Behaviour", behaviour)
+                            .WithField("Field", field.GetValue(behaviour)));
```

Comment:
What does `field.GetValue(...)` print for the user?

Reply:
This should (hopefully) call the default ToString() method of an object that was injected through the [Require] attribute.

Reply:
Wouldn't it be more useful to print the name of the field declared in the source if possible? So a user would get:

```
Field marked as required... 
Behaviour: MySpatialOSBehaviour
Field: MyIllegalRequireable
```

Reply:
You are right, that's much better. Fixed. :)